pkg/api/apis/operators/v1alpha2/operatorgroup_types.go

```diff
@@ -20,7 +20,13 @@ type OperatorGroupSpec struct {
 }
 
 type OperatorGroupStatus struct {
-	Namespaces  []string    `json:"namespaces,omitempty"`
+	// Namespaces is the set of target namespaces for the OperatorGroup.
+	Namespaces []string `json:"namespaces,omitempty"`
+
+	// ProvidedAPIs represents the set of APIs provided by the OperatorGroup's member CSVs.
+	// ProvidedAPIs []metav1.TypeMeta `json:"providedAPIs,omitempty"`
```

Comment:
I think this would be a valuable addition to the `status` block of the `OperatorGroup` CRD.

Reply:
I think so too, but it might be overkill for this fix, especially since none of the logic would use it for anything yet. I think we can phase this in with a follow up.